src/main/java/org/plumelib/javadoc/RequireJavadoc.java

```diff
@@ -289,6 +320,200 @@ boolean shouldExclude(Path path) {
     return shouldExclude(path.toString());
   }
 
+  /** The type of property method: a getter or setter. */
+  enum PropertyType {
+    /** A method of the form {@code int getFoo()}. */
+    GETTER("get", 0, false),
+    /** A method of the form {@code boolean hasFoo()}. */
+    GETTER_HAS("has", 0, false),
+    /** A method of the form {@code boolean isFoo()}. */
+    GETTER_IS("is", 0, false),
```

Comment:
I'm perfectly happy with `get`, `set`, and `is`.  However, since you introduced has, I'll just toss out there that `not` is sometimes common (the opposite of `is` for a boolean getter.)
```java
private boolean initialized = false;

public boolean notInitialized() {
  return !this.initialized;
}
```
I've also seen `unsetFoo()` to change a value to its default (or null).  Not suggesting you add that, either, just offering commentary for completeness.

Reply:
I added `not` since it was easy.  I'm not going to add `unset` -- at least, not now.

Reply:
Agreed, as in particular, "unset" is really "set to some empty or default thing" which needs a javadoc.